tests/fixture/mod.rs

```diff
@@ -14,28 +14,39 @@ use toml;
 
 const BUNDLE_OUT: &str = "worker";
 
-pub struct Fixture {
+pub struct Fixture<'a> {
     // we wrap the fixture's tempdir in a `ManuallyDrop` so that if a test
     // fails, its directory isn't deleted, and we have a chance to manually
     // inspect its state and figure out what is going on.
     dir: ManuallyDrop<TempDir>,
+    output_path: &'a str,
 }
 
-impl Default for Fixture {
+impl Default for Fixture<'_> {
     fn default() -> Self {
         Self::new()
     }
 }
 
-impl Fixture {
-    pub fn new() -> Fixture {
+impl Fixture<'_> {
+    pub fn new() -> Fixture<'static> {
         let dir = TempDir::new().unwrap();
         eprintln!("Created fixture at {}", dir.path().display());
         Fixture {
             dir: ManuallyDrop::new(dir),
+            output_path: BUNDLE_OUT,
         }
     }
 
+    pub fn new_site() -> Fixture<'static> {
+        let mut fixture = Fixture::new();
+        fixture.output_path = "dist";
```

Comment:
wouldn't this be `/workers-site/worker`?

Reply:
heh, we actually output to dist, also. but probably long term this is what we want to assert. will change.